js/util.js

```diff
@@ -7,7 +7,22 @@
  */
 
 import attributeClassNames from './common/util/attribute_class_names';
+import computeDomain from './common/util/compute_domain';
+import computeStateDisplay from './common/util/compute_state_display';
+import formatDate from './common/util/format_date';
+import formatDateTime from './common/util/format_date_time';
+import formatTime from './common/util/format_time';
 
 window.hassUtil = window.hassUtil || {};
 
+const language = navigator.languages ?
+  navigator.languages[0] : navigator.language || navigator.userLanguage;
+
+window.fecha.masks.haDateTime = window.fecha.masks.shortTime + ' ' + window.fecha.masks.mediumDate;
+
 window.hassUtil.attributeClassNames = attributeClassNames;
+window.hassUtil.computeDomain = computeDomain;
+window.hassUtil.computeStateDisplay = computeStateDisplay;
+window.hassUtil.formatDate = dateObj => formatDate(dateObj, language);
```

Comment:
Does this mean that we're not using the language preference? It's also not in the old method?

Reply:
That's right, at least that's how it was. I've updated state-card-display to pass in the language preference. As we convert these calls we should do the same, so that we can eventually get rid of this extra step.